src/main/groovy/org/edx/jenkins/dsl/AnalyticsConstants.groovy

```diff
@@ -164,7 +164,7 @@ This text may reference other parameters in the task as shell variables, e.g.  $
 
     public static def common_publishers = { allVars ->
         return {
-            mailer('$NOTIFY')
+            mailer(allVars.get('NOTIFY','$PAGER_NOTIFY'))
```

Comment:
@pwnage101 @brianhw this change ignores the NOTIFY parameter altogether, note the lack of `$` with NOTIFY. 
We could still do test runs of jobs by overriding the RECIPIENT section of the email notification post build action, not ideal. 
Open to suggestions

Reply:
I don't think we can check if the `NOTIFY` parameter is defined in this context.

Reply:
So this isn't quite doing what we want.  Another way to go would be to make sure that $NOTIFY is always set, and that it's default is set to $PAGER_NOTIFY.

Reply:
What do we want ? :) 
We have a default, $PAGER_NOTIFY, and we can override by defining `NOTIFY` in the extra vars files. 

What is missing ?

Reply:
Also, do we (not) have cases where the NOTIFY value is not in allVars, but rather in a more environment-specific one?   I'm thinking of the microbachelor DBT builds in particular.

Reply:
Okay, just saw them. I think you're referring to the ones here: https://github.com/edx-ops/analytics-secure/blob/master/job-configs/WAREHOUSE_TRANSFORMS_JOB_EXTRA_VARS.yaml

We could have it like
```
mailer(env.get('NOTIFY', allVars.get('NOTIFY', '$PAGER_NOTIFY')))
```

Reply:
as @brianhw mentioned in chat, we could use the flexible publish plugin, but this should work just fine. We already do it in some other jobs

Reply:
So `mailer(env.get('NOTIFY', allVars.get('NOTIFY', '$PAGER_NOTIFY')))` would be passing two arguments to mailer().   I'm not clear what gets into the $NOTIFY, and what is currently being loaded into vars.get('NOTIFY').   And if nothing is in allvars.get('NOTIFY'), then this would be adding $PAGER_NOTIFY, regardless of what is in $NOTIFY?

Reply:
Taking this to slack.